client/source/js/modules/costfunction/costfunction.js

```diff
@@ -520,7 +520,7 @@ define(['angular', 'underscore', 'toastr'], function(angular, _) {
           vm.outcomes.push({
             name: vm.state.parameter.short,
             pop: pop,
-            interact: "random",
+            interact: "nested",
```

Comment:
Does this make nested the default?

Reply:
@robynstuart thx for the eagle eyes, this was a debugging thing i forgot to take out!